pageserver/src/tenant.rs

```diff
@@ -1031,7 +1031,7 @@ impl Tenant {
         }
 
         Ok(TenantPreload {
-            timelines: Self::load_timeline_metadata(
+            timelines: Self::load_timelines_metadata(
```

Comment:
Hmm... Is this `Self::load_timelines_metadata(self,...)` even required as opposed to `self.load_timelines_metadata`?

Reply:
Hehe, I already compared `self` to the "inner" function `self: Arc<Tenant>` as it is the one below due to the wonky order.

Reply:
changed it to `self.load_timelines_metadata`